src/components/card-link/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from 'react'
+import classNames from 'classnames'
+import Card from 'components/card'
+import MaybeInternalLink from 'components/maybe-internal-link'
+import { CardLinkProps } from './types'
+import s from './card-link.module.css'
+
+const CardLink = ({
+  children,
+  className,
+  href,
+}: CardLinkProps): ReactElement => {
+  const classes = classNames(s.root, className)
+
+  return (
+    <MaybeInternalLink href={href}>
```

Comment:
At least in swingset, I'm seeing the anchor element expand to fit the full width of its containing element, so you can click in the empty space to the right of the card and you'll hit the link.

Reply:
Uh oh! 😨  Will get right on that.

Reply:
Just pushed a fix in https://github.com/hashicorp/dev-portal/pull/83/commits/6069a1fdf3b27ea5c9c3e4c97870554e2fc6f987!